packages/paste-core/components/combobox/src/styles/ComboboxListboxOption.tsx

```diff
@@ -1,74 +1,95 @@
   startHeight?: VirtualItem['start'];
 }
 
 const VariantStyles: {[key in ComboboxListboxOptionProps['variant']]: BoxStyleProps} = {
   groupOption: {
-    paddingLeft: 'space90',
-    paddingRight: 'space90',
+    paddingLeft: 'space70',
+    paddingRight: 'space50',
   },
   default: {
-    paddingLeft: 'space70',
-    paddingRight: 'space70',
+    paddingLeft: 'space50',
+    paddingRight: 'space50',
   },
 };
 
+const getVirtualStyles = (startHeight: number): Record<string, unknown> => ({
+  position: 'absolute' as PositionOptions,
+  top: 0,
+  left: 0,
+  width: '100%',
+  transform: `translateY(${startHeight}px)`,
+});
+
 const ComboboxListboxOption = React.forwardRef<HTMLLIElement, ComboboxListboxOptionProps>(
-  ({children, element = 'COMBOBOX', highlighted, variant = 'default', startHeight, ...props}, ref) => {
-    const virtualItemStyles =
-      startHeight !== undefined
-        ? {
-            position: 'absolute' as PositionOptions,
-            top: 0,
-            left: 0,
-            width: '100%',
-            transform: `translateY(${startHeight}px)`,
-          }
-        : {};
+  (
+    {
+      children,
+      element = 'COMBOBOX',
+      highlighted,
+      selected,
+      disabled = false,
+      variant = 'default',
+      startHeight,
+      ...props
+    },
+    ref
+  ) => {
+    const virtualItemStyles = startHeight != null ? getVirtualStyles(startHeight) : {};
+
     return (
       <Box
         {...safelySpreadBoxProps(props)}
         as="li"
+        disabled={disabled}
         element={`${element}_LIST_ITEM`}
-        backgroundColor={highlighted ? 'colorBackgroundPrimaryWeakest' : 'colorBackgroundBody'}
-        padding="space30"
-        cursor="pointer"
+        backgroundColor={highlighted && !disabled ? 'colorBackgroundPrimaryWeakest' : 'colorBackgroundBody'}
+        color={highlighted ? 'colorTextLinkStronger' : selected ? 'colorTextLink' : 'colorText'}
+        opacity={disabled ? 0.6 : 1}
+        display="flex"
+        alignItems="center"
+        paddingY="space40"
+        cursor={disabled ? 'not-allowed' : 'pointer'}
         ref={ref}
+        borderLeftWidth="borderWidth30"
+        borderLeftColor={highlighted && !disabled ? 'colorBorderPrimaryStronger' : 'transparent'}
+        borderLeftStyle="solid"
+        transition="borderLeft 200ms ease-in, backgroundColor 200ms ease-in"
+        borderBottomWidth="borderWidth10"
+        borderBottomColor="colorBorderDecorative10Weaker"
+        borderBottomStyle="solid"
+        _last={{
+          marginBottom: 'space0',
+          borderBottomColor: 'transparent',
+        }}
         {...virtualItemStyles}
         {...VariantStyles[variant]}
       >
-        <Text
-          as="span"
-          color="colorText"
-          element={`${element}_LIST_ITEM_TEXT`}
-          textDecoration={highlighted ? 'underline' : null}
-        >
-          {children}
-        </Text>
+        <Box as="span" minHeight="1.25rem">
```

Comment:
question: why do we need to give this a `minHeight`? is it because we've added an icon?